test/api/cli/io/WriteTest.cpp

```diff
@@ -21,9 +21,49 @@
 #include <catch.hpp>
 
 #include <filesystem>
+#include <fstream>
+#include <iostream>
 #include <string>
 
 const std::string dirPath = "test/api/cli/io/";
+const std::string dirPath2 = "test/runtime/local/io/";
+
+bool compareFiles(const std::string &filePath1, const std::string &filePath2) {
```

Comment:
Maybe this one could go into Utils.h? Ideally with a documenting comment (doxygen format) above the function signature ;-)